modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -0,0 +1,128 @@
+[id="cnf-assigning-a-secondary-network-to-a-vrf_{context}"]
+= Assigning a secondary network to a VRF
+
+[IMPORTANT]
+====
+The VRF feature described here is for Tech Preview purposes only and is not supported by Red Hat at this time.
+====
+
+As a cluster administrator, you can configure an additional network for your VRF domain by using the VRF CNI plug-in. The virtual network created by this plug-in is associated with a physical interface that you specify.
+
+[id="cnf-creating-an-additional-network-attachment-with-the-vrf-cni-plug-in_{context}"]
+== Creating an additional network attachment with the VRF CNI plug-in
+
+The Cluster Network Operator (CNO) manages additional network definitions. When you specify an additional network to create, the CNO creates the NetworkAttachmentDefinition custom resource (CR) automatically.
+
+[NOTE]
+====
+Do not edit the NetworkAttachmentDefinition CRs that the Cluster Network Operator manages. Doing so might disrupt network traffic on your additional network.
+====
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the OpenShift cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the CNO CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+. Extend the CR that you are creating by adding the `rawCNIConfig` configuration for the additional network, as in the example CR below. The following YAML configures the VRF CNI plug-in:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+  spec:
+  additionalNetworks:
+  - name: test-network-1
+    namespace: test-1
+    type: Raw
+    rawCNIConfig: '{
+      "cniVersion": "0.3.1",
+      "name": "macvlan-vrf",
+      "plugins": [  <1>
+      {
+        "type": "macvlan",  <2>
+        "master": "eth1",
+        "ipam": {
+            "type": "static",
+            "addresses": [
+            {
+                "address": "191.168.1.23/24"
+            }
+            ]
+        }
+      },
+      {
+        "type": "vrf",
+        "vrfname": "example-vrf-name",  <3>
+        "table": 1001   <4>
+      }]
+    }'
+----
+<1> `plugins` must be a list. The first item in the list must be the network plugin configuration, for example, `macvlan`. The second item in the list is the vrf plugin configuration.
```

Comment:
I think this could be made clearer by saying that the first item on the list must be the secondary network underpinning the VFR

Reply:
Thanks @djuran - I made this change. let me knoe if you spot anything else. This PR is due for merge any day now.